src/App.tsx

```diff
@@ -26,70 +27,96 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  if (sortType === 1) {
+    visibleGoods.sort();
+  }
 
-  return visibleGoods;
+  if (sortType === 2) {
+    visibleGoods.sort((a, b) => a.length - b.length);
+  }
```

Comment:
use switch. and names of types instead of indexes to make the code better to read